plugins/hm-stack/endpoint.php

```diff
@@ -0,0 +1,238 @@
+<?php
+/**
+ * Build endpoint data for HM Stack data on this site.
+ *
+ * @package HMStackIntegration
+ */
+
+namespace HM_Stack\Endpoint;
```

Comment:
Should these files not be `hm-stack/inc/endpoint/namespace.php`? cc @rmccue

Reply:
Erm, I'd prefer to not use that naming convention but will modify if you guys really would like.

P.S. I've actually been meaning to write up a post/question on this - not trying to be difficult 😬

Reply:
Yep, it should be. I've also been planning about writing up a post for feedback on this standard anyway :)

Reply:
Cool cool, got some consensus here so I'll update this 👍